packages/panzer/adapters-stk/example/PoissonExampleCurvilinear/compareWithGold.py

```diff
@@ -0,0 +1,64 @@
+import math
+import sys
+import re
+
+try:
+  if not len(sys.argv)==5:
+    print('Expected four files, two gold and the two test outputs')
+    raise Exception("Failure: Command line - <cmd> [test_Q2_mesh.gold] [test_Q1_mesh.gold] [test_Q2_mesh.out] [test_Q1_mesh.out]")
+
+  q2gold = sys.argv[1]
+  q1gold = sys.argv[2]
+  q2test = sys.argv[3]
+  q1test = sys.argv[4]
+
+  ErrorPat = re.compile('.*Error = (.*)$')
+
+  q2gold_errors = []
+  q1gold_errors = []
+  q2_errors = []
+  q1_errors = []
+
+  with open(q2gold) as f:
+    for line in f:
+      match = ErrorPat.match(line)
+      if match!=None:
+        q2gold_errors.append(float(match.group(1)))
+  with open(q1gold) as f:
+    for line in f:
+      match = ErrorPat.match(line)
+      if match!=None:
+        q1gold_errors.append(float(match.group(1)))
+  with open(q2test) as f:
+    for line in f:
+      match = ErrorPat.match(line)
+      if match!=None:
+        q2_errors.append(float(match.group(1)))
+  with open(q1test) as f:
+    for line in f:
+      match = ErrorPat.match(line)
+      if match!=None:
+        q1_errors.append(float(match.group(1)))
+
+  for i in range(len(q2gold_errors)):
+    if q2gold_errors[i] != q2_errors[i]:
```

Comment:
I think these comparisons of floating point numbers should use a tolerance. What is the expected accuracy?

Reply:
This is a straight regression test -- I'll add a tolerance of 1e-14.